providers/flagd/pkg/service/grpc/tests/service-resolve-boolean_test.go

```diff
@@ -0,0 +1,154 @@
+package tests
```

Comment:
What's the reason for moving tests into their own directory? I've not seen this approach before, [generally tests are included in the same dir](https://go.dev/doc/tutorial/add-a-test) as what they're testing with the same package name appended with `_test`. It'd be good to remain consistent

Reply:
updated to have tests in the same directory as the corresponding services

Reply:
imo this still looks a bit unfamiliar. Since you are basically just testing the logic/methods of the `HTTP/GRPCService` that are living inside the `service.go` files,  I would propose to move all the tests to a corresponding `service_test.go` file.

Reply:
updated all service tests to sit within the `service_test.go` file(s)